docs/source/interfaces/DrILL.rst

```diff
@@ -131,6 +131,40 @@ tooltip (visible when the mouse moves over the cell) explains the error. A
 single red cell prevent the processing of the concerned row.
 
 
+Groups
+------
+
+To avoid entering exactly the same value several times in the table, it is also
+possible to create groups of samples. Within a group, a master sample can be
+designated. The values of the parameters of the master sample will be used when
+processinga all rows in the group.
+
+Paramaters can still be overriden manually whithin a group by entering a sample
+specific value in the table. The priority for the parameter values is as follow:
+
+sample > master sample > global settings
+
+Example:
+
+=========  =====  ===========  ===========
+Sample     Group  parameter 1  parameter 2
+=========  =====  ===========  ===========
+1(master)  g1     v1           v2
+2          g1
+3          g1                  v2'
+=========  =====  ===========  ===========
```

Comment:
I think we should also be able to handle a case, when for a slave row we will need to disable a given correction. In the example above, what if for sample 2 we don't want to use the v1 as parameter1 value. Is there an easy way of doing this? Of course, one could manually write the default value of the parameter in order overwrite it. But for some corrections the default is actually empty. Maybe we should have a symbol for deliberately empty cells.